EIPS/eip-custom_errors_for_erc_tokens.md

```diff
@@ -0,0 +1,488 @@
+---
+eip: [assignedEIPNumber]
+title: Custom errors for ERC tokens
+description: List of Solidity custom errors covering standard token implementations (EIP-20, EIP-721 and EIP-1155)
+author: Ernesto García (@ernestognw), Francisco Giordano (@frangio), Hadrien Croubois (@Amxx)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: ERC
+requires: 20, 721, 1155
+created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
+---
+
+## Abstract
+
+The following specification allows for the use of a standard list of Solidity [custom errors](https://blog.soliditylang.org/2021/04/21/custom-errors/) to be used within [EIP-20](./eip-20.md), [EIP-721](./eip-721.md) and [EIP-1155](./eip-1155.md) tokens.
```

Comment:
EIP editors will ask us to remove all external links. https://eips.ethereum.org/EIPS/eip-1#linking-to-external-resources

Generally the policy is:

>  If it is possible to implement a standard without a linked resource, then the linked resource is unnecessary.

Reply:
I removed all of the external links.
Do any of them sound required to you? So I can recover it.